lib/nostrum/consumer.ex

```diff
@@ -111,6 +112,7 @@ defmodule Nostrum.Consumer do
   @type user_update :: {:USER_UPDATE, {old_user :: Nostrum.Struct.User.t, new_user :: Nostrum.Struct.User.t}, ws_state}
   @type voice_state_update :: {:VOICE_STATE_UPDATE, {map}, ws_state}
   @type voice_server_update :: {:VOICE_SERVER_UPDATE, {map}, ws_state}
+  @type webhooks_update :: {:WEBHOOKS_UPDATE, {webhook :: Nostrum.Struct.Webhook.t}, ws_state}
```

Comment:
Could we leave the return type here as a map? Most of these types are lies anyways atm, but since Nostrum doesn't track webhooks at all currently its even further from the truth,